DATA/UNIVERSE/SYSTEMS/Br01/Br01.ini

```diff
@@ -10289,3 +10289,215 @@ parent = Br01_08
 visit = 128
 Archetype = jump_hole_invisible
 goto = Br01, br01_from_start_unlawful, gate_tunnel_bretonia
+
+;;Mineable Gas Pockets
+
+[Object]
+nickname = mineable_small_co2_Br01_01
+ids_name = 459177
+;res str
+; CO2 Gas Pocket
+ids_info = 460651
+;res html
+; \m\bCO2 Gas Pocket\l\B
+; 
+; A and dense gaseous mass of CO2, can only be harvested with the appropriate equipment.\n
+pos = 16131.3988, 0.000, 79347.9212
+archetype = mineable_gas_pocket
+phantom_physics = true

```

Comment:
Phantom Physics must be only in the solararch.ini. It does nothing in the systems.

Reply:
Nope, had it just in solararch and bumbed against the invisible object. It needs to be in the system as well

Reply:
This cannot be true, or every jumphole in the game would have an actual collision.